.github/workflows/notify-on-force-push.yml

```diff
@@ -0,0 +1,29 @@
+name: Notify on Push
+
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: read
+
+jobs:
+  notify-on-push:
+    name: Notify on Force Push to `main`
+    if: github.repository == 'nodejs/release-cloudflare-worker' && github.event.forced
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Slack Notification
+        uses: rtCamp/action-slack-notify@c33737706dea87cd7784c687dadc9adf1be59990 # 2.3.2
+        env:
+          SLACK_COLOR: '#DE512A'
+          SLACK_ICON: https://github.com/nodejs.png?size=48
+          SLACK_TITLE: ${{ github.actor }} force-pushed to ${{ github.ref }}
+          SLACK_MESSAGE: |
+            A commit was force-pushed to <https://github.com/${{ github.repository }}/tree/${{ github.ref_name }}|${{ github.repository }}@${{ github.ref_name }}> by <https://github.com/${{ github.actor }}|${{ github.actor }}>
+            Before: <https://github.com/${{ github.repository }}/commit/${{ github.event.before }}|${{ github.event.before }}>
+            After: <https://github.com/${{ github.repository }}/commit/${{ github.event.after }}|${{ github.event.after }}>
+          SLACK_USERNAME: nodejs-bot
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
```

Comment:
Need to generate this

Reply:
Imo the webhook should be for the `nodejs-web-infra-alerts` channel in Slack

Reply:
@flakey5 let me give you permissions.

Reply:
I've given you admin permissions.